indico/modules/users/models/users.py

```diff
@@ -783,6 +784,13 @@ def get_merged_from_users_recursive(self):
         cte = cte_query.union_all(rec_query)
         return User.query.join(cte, User.id == cte.c.merged_from_id).all()
 
+    @contextmanager
+    def force_user_locale(self):
+        """Temporarily override the locale to the locale of the user."""
+        locale = self.settings.get('lang') or None
```

Comment:
do we sometimes have a falsy value there that's not `None`? if not the `or None` isn't needed :)